adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/CoapContext.java

```diff
@@ -307,4 +308,13 @@ private Integer getIntegerQueryParameter(final String parameterName) {
                 .orElse(null);
     }
 
+    /**
+     * Get the QoS level as set in the request by the device.
```

Comment:
Can you please always use present tense active form? In this case
> Gets the quality of service level corresponding to the delivery semantics specified by the device in its request to upload data.

This applies to most (all) of your JavaDoc comments ...

Reply:
Will do.